src/floats.jl

```diff
@@ -500,7 +509,8 @@ end
             (Ref{BigFloat}, Ref{BigFloat}, Ref{BigFloat}, Int32),
             x, x, y, MPFR.ROUNDING_MODE[])
     end
-    return T(neg ? -x : x)
+    # See https://github.com/JuliaData/CSV.jl/issues/938
+    return neg ? T(-x) : unalias_bigfloat(T, x)
```

Comment:
Ah, interesting. It doesn't affect negative BigFloats, because a new BigFloat gets allocated and :mpfr_neg gets called. Sheesh.

Reply:
Yes :)